packages/extensions-sdk/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@directus/extensions-sdk",
-	"version": "12.1.4",
+	"version": "12.2.0",
```

Comment:
We don't need to bump the version manually, rather just add a change set for this package by running `changeset` in the project root